src/odemis/acq/stream/_helper.py

```diff
@@ -1451,6 +1451,36 @@ def __init__(self, name, detector, dataflow, emitter, **kwargs):
             kwargs["acq_type"] = model.MD_AT_EM
         super().__init__(name, detector, dataflow, emitter, **kwargs)
 
+
+class DigitalEBICStream(FastScanningDetector):
+    """
+    A special EBIC stream, typically with a EBIC (current) as a detector and its own scanner.
+    It's physically very similar to the SEM stream, but as we want to select just a region
+    it's in practice similar to the CLSettingStream.
+
+    In live view, the ROI is not applied, but the pixelSize is.
+
+    This stream makes it possible to acquire an image simultaneously with the
+    SED in live view, but as a requirement they would need to align dwell time/resolution.
+    """
+    def __init__(self, name, detector, dataflow, emitter, **kwargs):
+        """
+        emtvas: don't put resolution or scale
+        """
+        if "acq_type" not in kwargs:
+            kwargs["acq_type"] = model.MD_AT_EBIC
+        super().__init__(name, detector, dataflow, emitter, **kwargs)
+
+        self._ebic = detector
+
+        self.repetition.subscribe(self.onRepetitionChange)
+        # if model.hasVA(self._detector, "spp"):
+        #     self.spp = model.IntContinuous(self._detector.spp.value, (1, 10))
+        #     self.spp.subscribe(self._onSppChange)
+
+    def onRepetitionChange(self, value):
```

Comment:
I think that the main part missing here is in live mode. The hardware is normally configured so that the SPARC scanner (only) controls the e-beam. So if we want to acquire an image, we have to acquire simultaneously a EBIC data, and run the "e-beam" component. This is typically done by acquiring on the se-detector. 
We need special code that handle this.